typescript/sdk/src/utils/chains.ts

```diff
@@ -0,0 +1,7 @@
+import type { ChainMap } from '../types';
+
+export function inferChainMap<M>(map: M) {
```

Comment:
I don't think this is actually used/useful

Reply:
Me neither! Killing it.